test/objects/linode/linode_test.py

```diff
@@ -89,3 +89,89 @@ def test_available_backups(self):
         # assert that snapshots came back as expected
         self.assertEqual(backups.snapshot.current, None)
         self.assertEqual(backups.snapshot.in_progress, None)
+
+    def test_create_linode(self):
+        """
+        Tests that a Linode can be created
+        """
+        with self.mock_post('linode/instances/123') as m:
+            linode = Linode(self.client, 123)
+
+            self.assertIsNotNone(linode)
+            self.assertEqual(linode.id, 123)
+            self.assertEqual(linode.label, "linode123")
+            self.assertEqual(linode.group, "test")
+            self.assertTrue(isinstance(linode.image, Image))
+            self.assertEqual(linode.image.label, "Ubuntu 17.04")
```

Comment:
This isn't testing Linode creation, this is getting a Linode by ID.

Reply:
Apparently there is already a test in linode_client_test.py